tests/vasp/lobster/flows/test_lobster.py

```diff
@@ -141,11 +144,14 @@ def test_lobstermaker(mock_vasp, mock_lobster, clean_dir, memory_jobstore):
     job = VaspLobsterMaker(
         lobster_static_maker=LobsterStaticMaker(),
         lobster_maker=LobsterMaker(
+            task_document_kwargs={
+                "calc_quality_kwargs": {"potcar_symbols": ["Si"], "n_bins": 10}
+            },
```

Comment:
Is this a fix for the test?

Reply:
Yes, no POTCARS and very low res lobster computations